hawc/apps/epiv2/admin.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib import admin
+
+from . import models
+
+
+class AdjustmentFactorInline(admin.TabularInline):
+    model = models.AdjustmentFactor
+    extra = 0
+
+
+class ChemicalInline(admin.TabularInline):
+    model = models.Chemical
+    extra = 0
+
+
+class CriteriaInline(admin.TabularInline):
+    model = models.Criteria
+    extra = 0
+
+
+class DataExtractionInline(admin.TabularInline):
+    model = models.DataExtraction
+    extra = 0
+
+
+class ExposureInline(admin.TabularInline):
+    model = models.Exposure
+    extra = 0
+
+
+class ExposureLevelInline(admin.TabularInline):
+    model = models.ExposureLevel
+    extra = 0
+
+
+class OutcomeInline(admin.TabularInline):
+    model = models.Outcome
+    extra = 0
+
+
+class StudyPopulationAdmin(admin.ModelAdmin):
+    list_display = (
```

Comment:
add ability to search too maybe by name and study__short_citation for now if we can

Also add some list_filters for some of the metadata on this

Reply:
Can you explain the list_filters comment a bit?